src/hct_mis_api/contrib/aurora/migrations/0003_migration.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.2.25 on 2024-11-29 13:55
+
```

Comment:
you also have to add this migration into Jan's migrations script `excluded_migrations `

https://github.com/unicef/hope/blob/33609dee2abfa8fa9ca5534502890a0e538dc5b9/src/hct_mis_api/migrations_script/main.py#L48

Reply:
👍🏻